public/app/features/alerting/unified/components/rule-viewer/Actions.tsx

```diff
@@ -71,6 +73,19 @@ export const useAlertRulePageActions = ({ handleDelete, handleDuplicateRule }: P
               childItems={[<ExportMenuItem key="export-with-modifications" identifier={identifier} />]}
             />
           )}
+          {rulePluginLinkExtension.length > 0 && (
+            <>
+              <Menu.Divider />
+              {rulePluginLinkExtension.map((extension) => (
+                <Menu.Item
+                  key={extension.id}
+                  label={extension.title}
+                  icon={extension.icon}
+                  onClick={extension.onClick}
```

Comment:
I think a `PluginExtensionLink` optionally has `onClick`, and can have a `path` - should we handle that here?

And possibly consider what we do when there's neither of those properties? Though I'd imagine we should have one or the other, perhaps we don't show it when both are omitted

Reply:
Yes I think we should. 
Regarding what we do when there's neither of those properties - this is already handled by the plugin framework
If an extensions doesn't provide onClick nor path it won't be returned from the `getPluginLinkExtensions` function

Reply:
Exactly as @konrad147 says, we filter out the misconfigured link extensions (which don't have either a path or an onclick defined).

Reply:
I think you just can add `url={extension.path}` and it should be covered.